Sources/Apodini/Semantic Model Builder/OpenAPI/Document/OpenAPIComponentsObjectBuilder.swift

```diff
@@ -27,69 +28,91 @@ class OpenAPIComponentsObjectBuilder {
         vendorExtensions: [:]
     )
 
-    /// In case more than one type in HTTP body, build wrapper schema.
+    /// Use this function to build a schema for an arbitrary type.
+    /// Types built with function are automatically stored into the componentsObject
+    /// and thus can be referenced throughout the specification.
+    /// In case, the given type is a reference type, the reference to the schema will be returned.
+    func buildSchema(for type: Encodable.Type) throws -> JSONSchema {
+        let (schema, _) = try buildSchemaWithTitle(for: type)
+        return schema
+    }
+
+    /// For responses, we create wrapper objects as returned by the REST API.
+    /// Therefore we use the `ResponseContainer`'s CodingKeys
```

Comment:
Although I get the intention, I'd suggest to not use `we` here. Since these comments are publicly exposed as doc, they should not contain internal explanation.

Reply:
Alright, I will fix this throughout the file! 👍

Reply:
solved ee1995e